src/components/RelatedVideoList/RelatedVideoCard/RelatedVideoCard.spec.jsx

```diff
@@ -0,0 +1,119 @@
+                channelTitle="channel Title"
+                goToVideoHandler={mockedGoToVideo}
+                thumbUrl="image"
+              />
+            </ThemeProvider>
+          </DarkModeProvider>
+        </BrowserRouter>
+      );
+
+      expect(getByText(/channel title/i)).toBeInTheDocument();
+    });
+
+    test('renders when props.author is null', () => {
+      const video = {};
+      const { getByText } = render(
+        <BrowserRouter>
+          <DarkModeProvider>
+            <ThemeProvider>
+              <RelatedVideoCard
+                goToVideoHandler={mockedGoToVideo}
+                thumbUrl="image"
+                title="title"
+                channelTitle={video.author}
+              />
+            </ThemeProvider>
+          </DarkModeProvider>
+        </BrowserRouter>
+      );
+      expect(getByText(/title/i)).toBeInTheDocument();
+    });
+  });
+
+  // since sometimes API returns coded strings
+  describe('RelatedVideoCard shown texts are decoded', () => {
+    const mockedGoToVideo = jest.fn(() => {});
+
+    const codedString = 'Wizeline&#39;s';
+    test('title gets decoded', () => {
+      const { getByText } = render(
+        <BrowserRouter>
+          <DarkModeProvider>
+            <ThemeProvider>
+              <RelatedVideoCard
+                goToVideoHandler={mockedGoToVideo}
+                thumbUrl="image"
+                title={codedString}
+                channelTitle="author"
+              />
+            </ThemeProvider>
+          </DarkModeProvider>
+        </BrowserRouter>
+      );
+      expect(getByText(/Wizeline's/i)).toBeInTheDocument();
+    });
+
+    test('channelTitle gets decoded', () => {
+      const { getByText } = render(
+        <BrowserRouter>
+          <DarkModeProvider>
+            <ThemeProvider>
+              <RelatedVideoCard
+                goToVideoHandler={mockedGoToVideo}
+                thumbUrl="image"
+                title="title"
+                channelTitle={codedString}
+              />
+            </ThemeProvider>
+          </DarkModeProvider>
+        </BrowserRouter>
+      );
+      expect(getByText(/Wizeline's/i)).toBeInTheDocument();
+    });
+  });
+
+  describe('Testing  onClick handler', () => {
+    const mockedGoToVideo = jest.fn(() => {});
+
+    it('calls goToVideoHandler when VideoCard is clicked', () => {
+      const { getByText } = render(
+        <BrowserRouter>
+          <DarkModeProvider>
+            <ThemeProvider>
+              <RelatedVideoCard
+                goToVideoHandler={mockedGoToVideo}
+                thumbUrl="image"
+                title="video title"
+                channelTitle="author"
+              />
+            </ThemeProvider>
+          </DarkModeProvider>
+        </BrowserRouter>
+      );
+
+      const videoCard = getByText(/title/i);
+      fireEvent.click(videoCard);
+      expect(mockedGoToVideo).toHaveBeenCalledTimes(1);
+    });
+  });
+});
```

Comment:
Nice testing. An edge case I suggest you to test is what happens with your default props. Try testing the component with only the required properties and see if it behaves as expected. Perhaps you'll find that you need to require another one.